src/main/java/org/opendatakit/aggregate/form/Form.java

```diff
@@ -713,5 +713,8 @@ public String getUri() {
     return infoRow.getUri();
   }
 
-
+  @Override
+  public boolean isValid() {
+    return filesetRow.hasField(FormInfoFilesetTable.IS_DOWNLOAD_ALLOWED);

```

Comment:
Why was this chosen as the measure of validity? Is it the last thing that is set before a form becomes available? Or rather, is it possible for a form to be invalid after this stage?

Reply:
To be honest, it's just because that field being null is what threw an `NPE`, producing the refresh hell.

Reply:
This is just a temporary fix while we work on #134